lib/inferno/dsl/fhir_evaluator/dataset.rb

```diff
@@ -0,0 +1,38 @@
+module Inferno
+  module DSL
+    module FHIREvaluation
+      # A Dataset returns an Array of FHIR data to be summarized or evaluated,
+      # with convenience methods for loading from a file path or from an
+      # Array of FHIR JSON strings.
+      module DatasetLoader
```

Comment:
From discussion - update filename to match module name

Reply:
Updated.